.github/autobuild/get_build_vars.py

```diff
@@ -32,36 +32,24 @@ def get_git_hash():
     ]).decode('ascii').strip()
 
 
-def write_changelog(version):
-    changelog = subprocess.check_output([
-        'perl',
-        f'{REPO_PATH}/.github/actions_scripts/getChangelog.pl',
-        f'{REPO_PATH}/ChangeLog',
-        version,
-    ])
-    with open(f'{REPO_PATH}/autoLatestChangelog.md', 'wb') as f:
-        f.write(changelog)
-
-
-def get_build_version(jamulus_pro_version):
+def get_build_version():
+    jamulus_pro_version = get_version_from_jamulus_pro()
     if "dev" in jamulus_pro_version:
```

Comment:
I'm not keen on logging in this function.  I'd rather it returned a tuple of the type and version and the caller could choose to log or not.
```
type = "release"
version = jamulus_pro_version
if "dev" in jamulus_pro_version:
    type = "dev"
    version = "{}-{}".format(version, get_git_hash())
return type, version
```
```
type, version = get_build_version()
print(f"building a {type} version: {version}")
```

Reply:
I don't feel strongly about that and I think I've just taken what was there.
I have now (mostly) applied your suggestion.

Reply:
Yep wasn't fussed as it was already there.  But as it was potentially useful information (i.e. build type), it might as well be exposed to the caller.

Reply:
> On which note - is the check for `dev` here the consistent check now?

Sorry, I missed that part -- Github doesn't send notifications for edits.
Can you elaborate on what you mean here?